types/coin.go

```diff
@@ -576,8 +591,8 @@ func (coins Coins) Sort() Coins {
 // Parsing
 
 var (
-	// Denominations can be 3 ~ 64 characters long.
-	reDnmString = `[a-z][a-z0-9/]{2,63}`
+	// Denominations can be 3 ~ 128 characters long.
+	reDnmString = `[a-zA-Z][a-zA-Z0-9/]{2,127}`
```

Comment:
Since we're making a standalone PR for this, is there a reason we shouldn't use unicode? i.e. `\p{Letter}` and `\p{Number}`. We are we excluding `_`, `-`, and `.`?

Reply:
done